### DISCUSSION TITLE
Component: Input Number - comma in NumpadDecimal

### DISCUSSION THREAD

**Discussion by ajjp** [primefaces/community#784] [PrimeNG]
Be able to define or automatically determine the behavior of the NumpadDecimal key to emit a period or a comma depending on the value of the locale.

i.e. if the locale uses a comma as a decimal point then the emitted NumpadDecimal value should be a comma, otherwise a period.

This has been requested by the users because it's very useful when they are using the numeric keypad to enter many decimal values in a row as it would make possible to use just that part of the keyboard.

![254025468-b436f0a6-d825-4747-9633-64d07a3e3937](https://github.com/primefaces/community/assets/106910824/bae4a5dd-0c78-4551-b8fc-09f1cb48e0aa)
![254025610-ea56054b-caa1-4708-8394-da48c7023499](https://github.com/primefaces/community/assets/106910824/522e80d4-cd25-4f31-935a-047b3d45470b)

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we've added it in our internal backlog.
